language.json

```diff
@@ -7,6 +7,18 @@
         "loggedIn": "Logged in as %{name}",
         "logout": "Logout",
         "collection": "Collection",
+        "collections": {
+          "Alger": "Algae",
```

Comment:
U should actually check if there is a translation for the name. If there is no translation use the name without translating it.